js/login.js

```diff
@@ -0,0 +1,48 @@
+// 아이콘(눈) 버튼 클릭 시 비밀번호 보이기 / 숨기기
+let eye = document.querySelector('.eye');
+let password = document.querySelector('#password-input');
+
+function passwordShow () {
```

Comment:
기본적으로 로직은 잘 작성해 주셨습니다. 다만 함수명 변수 설정할 땐 동사로 시작하도록 짓는게 좋습니다.
해당 함수의 경우 show 와 hide 를 모두 처리하고 있기 때문에 show 보다는 togglePasswordVisibility 가 더 명확해보입니다.

Reply:
로직 내부에서 password.setAttribute()가 두 번 반복되는데, 코드를 작성할 때 가급적 중복 요소는 줄이는게 좋습니다. 아래와 같이 줄여볼 수 있을 것 같네요. 또한 eye.classList도 toggle 을 사용하면 한 줄로 정리할 수 있습니다.
```suggestion
  const isPasswordHidden = password.getAttribute('type') === 'password';
  eye.classList.toggle('show', isPasswordHidden);  // 조건에 따라 class 토글
  password.setAttribute('type', isPasswordHidden ? 'text' : 'password'); 
```